src/main.rs

```diff
@@ -299,30 +299,65 @@ fn run(
         });
     }
 
+    let mut pressed_g = 'n';
```

Comment:
Also can you add a comment above this reading, like "tracking 'gg' key binding"?